similarity/pom.xml

```diff
@@ -16,12 +16,61 @@
             <artifactId>illinois-core-utilities</artifactId>
             <version>3.1.7</version>
```

Comment:
It's actually well-formatted in my IDE. I don't know why it look like this way in git.